.github/CONTRIBUTING.md

```diff
@@ -101,439 +108,6 @@ Authenticates and Uploads from Dropbox through Companion:
-`.env.example` to a file named `.env` and add the values relevant for the
-test(s) you are trying to run.
-
-To start the testing suite run:
-
-```
-yarn e2e
-```
-
-This will run Cypress in watch-mode, and it will pick up and rebuild any changes
-to JS files. If you need to change other files (like CSS for example), you need
-to run the respective `yarn build:*` scripts.
-
-Alternatively the following command is the same as the above, except it doesn’t
-run `build` first:
-
-```
-yarn e2e:skip-build
-```
-
-To generate the boilerplate for a new test run:
-
-```
-yarn e2e:generate
-```
-
-## Releases
-
-Releases are managed by GitHub Actions, here’s an overview of the process to
-release a new Uppy version:
-
-- Run `yarn release` on your local machine.
-- Follow the instructions and select what packages to release. **Warning:**
-  skipping packages results in those changes being “lost”, meaning they won’t be
-  picked up in the changelog automatically next release. Always try to release
-  all.
-- Before committing, check if the generated files look good.
-- When asked to edit the next CHANGELOG, only include changes related to the
-  package(s) you selected for release.
-- Push to the Transloadit repository using the command given by the tool. Do not
-  open a PR yourself, the GitHub Actions will create one and assign you to it.
-- Wait for all the GitHub Actions checks to pass. If one fails, try to figure
-  out why. Do not go ahead without consulting the rest of the team.
-- Review the PR thoroughly, and if everything looks good to you, approve the PR.
-  Do not merge it manually!
-- After the PR is automatically merged, the demos on transloadit.com should also
-  be updated. Check that some things work locally:
-  - the demos in the demo section work (try one that uses an import robot, and
-    one that you need to upload to)
-  - the demos on the homepage work and can import from Google Drive, Instagram,
-    Dropbox, etc.
-
-If you don’t have access to the transloadit.com source code ping @arturi or
-@goto-bus-stop and we’ll pick it up. :sparkles:
-
-### Fixing a broken release
-
-If the `Release` job fails, it will merge the release PR and deletes the branch
-and there’s no way of going back with that PR. Instead you have to reverse the
-release commit and run the Release candidate job again to create a new release
-PR:
-
-1. Checkout `main` locally and reverse the automatically release commit
-2. Do whatever changes need to be done on `main`
-3. Commit and push `main`
-4. Now checkout the `release` branch and rebase it onto `main`, then push it
-5. A new release PR will now be created and you can start over
-6. Delete tags for the original release commit (`COMMIT_ID`):
-
-```bash
-git tag --points-at COMMIT_ID | tee /dev/tty | xargs -n 1 git tag -d
-```
-
-7. Push the deleted tags:
-
-```bash
-git tag --points-at COMMIT_ID | xargs -n 1 -I {} echo ":refs/tags/{}" | xargs git push origin
-```
-
-### Releasing hotfix patch
```

Comment:
can we still do hotfixes on old versions? let's document it

Reply:
I would like the contributing.md to be designed for outside contributors, not as our personal knowledge base. Thing only we need to know shouldn't be here.